specfile/changelog.py

```diff
@@ -0,0 +1,134 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import collections
+import datetime
+from typing import List, Optional, Union
+
+import arrow
+
+from specfile.sections import Section
+
+
+class ChangelogEntry:
+    """
+     Class that represents a changelog entry.
+
+    Attributes:
+        header: Header of the entry.
+        content: List of lines forming the content of the entry.
+    """
+
+    def __init__(
+        self,
+        header: str,
+        content: List[str],
+    ) -> None:
+        self.header = header
+        self.content = content.copy()
+
+    def __str__(self) -> str:
+        return f"{self.header}\n" + "\n".join(self.content) + "\n"
+
+    def __repr__(self) -> str:
+        content = repr(self.content)
+        return f"ChangelogEntry('{self.header}', {content})"
+
+    @property
+    def extended_timestamp(self) -> bool:
+        """Whether the timestamp present in the entry header is extended (date and time)."""
+        try:
+            arrow.get(self.header, "ddd MMM DD YYYY")
```

Comment:
Are you sure that this works correctly? Since you are passing the whole header to `arrow.get` now, this will always throw an exception, won't it? Since `self.header` is the whole `* <datetime> <author> <NVR>` string.

Reply:
No, the `get()` method actually searches for a match within the whole string (admittedly it's not made obvious in the official documentation, apart from several code examples), so it works as expected.